app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,54 @@
+class TweetsController < ApplicationController
+   get '/tweets' do
+      logged_in? ? (erb :'tweets/tweets') : (redirect '/login')
+   end
+
+   get '/tweets/new' do
+      logged_in? ? (erb :'tweets/create_tweet') : (redirect '/login')
+   end
+
+   post '/tweets/new' do
```

Comment:
Typically, tweet creation would be posted to `/tweets` instead of `/tweets/new`.